### ISSUE TITLE
Integrate conference call

### ISSUE THREAD

**Issue by sualko** [jsxc/jsxc#3]
##

**Comment by poVoq** [comment]
Jitsi Meet uses a XMPP technology called COLIBI videobridge. See: https://xmpp.org/extensions/xep-0340.html

They have implemented as a standard xmpp component that by default connects to an internal Prosody server, but which can also work with Ejabberd.
There is also an new advanced version called videobridge2.

Would be really cool if colibri videobridging was supported by JSXC :)

**Comment by poVoq** [comment]
Their library would probably make it possible to use the videobridge quite easily?

See:
https://github.com/jitsi/lib-jitsi-meet

It is for example implemented as a plugin in Conversejs:
https://github.com/conversejs/community-plugins/blob/master/jitsimeet/readme.md

**Comment by sualko** [comment]
From the conversejs screenshot it looks more like there is a jitsi meet instance inside an iframe or so, but jitsi video bridge is definitely something we should consider.

**Comment by DanScharon** [comment]
yeah, the conversejs plugin doesn't seem to use lib-jitsi-meet or what am I missing here? 🤔️

**Comment by poVoq** [comment]
Sorry, I was confused. It is the Audioconf plugin that uses the jitsimeet library:
https://github.com/conversejs/community-plugins/tree/master/audioconf

**Comment by poVoq** [comment]
https://gist.github.com/iNPUTmice/a28c438d9bbf3f4a3d4c663ffaa224d9

Some notes on interoperability with Conversations.

**Comment by sualko** [comment]
https://dev.to/aminarria/scalable-broadcasting-using-webrtc-2984
https://doc-kurento.readthedocs.io/en/6.14.0/

[XEP-0272: Multiparty Jingle](https://xmpp.org/extensions/xep-0272.html)

**Comment by poVoq** [comment]
Since Dino.im supports SFU-less group calls now, maybe it would make sense to merge that part into the main branch at least?
